packages/docs/page-config/introduction/roadmap/images/1.6/autoComplete.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <img :src="Image"/>
+  <img :src="Image" alt="autoComplete"/>
```

Comment:
```suggestion
  <img :src="Image" alt="" />
```

It is also decorative image. Having extra alt text produces incorrect link text (easy to find in Firefox accessibility devtools):
<img width="530" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/23530004/5076c4bf-8e47-4c1a-8107-43cf9cc329e6">

Actually, all images on roadmap page (even svg) must have aria hidden or empty alt.